front/lib/plans/stripe.ts

```diff
@@ -30,13 +30,13 @@ async function getPriceId(productId: string): Promise<string | null> {
  * Once the users has completed the checkout, we will receive an event on our Stripe webhook
  */
 export const createCheckoutSession = async ({
-  owner,
+  workspaceSId,
```

Comment:
We generally call this `workspaceId` in the rest of the code

Reply:
But that being said the call site has access to the `owner: WorkspaceType` why not pass it, this cleaner IMHO?